helm-chart/kuberay-operator/templates/deployment.yaml

```diff
@@ -61,6 +69,18 @@ spec:
             {{- $argList = append $argList "--watch-namespace" -}}
             {{- $argList = append $argList $watchNamespace -}}
             {{- end -}}
+            {{- if and (.Values.logging.baseDir) (.Values.logging.fileName) -}}
```

Comment:
What will happen if users specify only one of `baseDir` or `fileName`?

Reply:
We don't render --log-file at all in this case